src/linagora.esn.contact/app/contact/action/copy/contact-action-copy.pug

```diff
@@ -1,6 +1,7 @@
-a(
-  href='',
+button.btn.btn-link.waves-effect(
```

Comment:
does it look good on the 3 dots action menu ( on the right side ) of the contact when listing an address book?

Reply:
Oh shit it doesn't. I'll fix it now. Nice catch.

![image](https://user-images.githubusercontent.com/24670327/112453280-1c9d3480-8d8a-11eb-837a-4315371eb44d.png)

Reply:
It looks good now:

![image](https://user-images.githubusercontent.com/24670327/112456770-a6023600-8d8d-11eb-82eb-bc86ef067a99.png)

![image](https://user-images.githubusercontent.com/24670327/112456810-aef30780-8d8d-11eb-971b-48c9e890e3c7.png)